students/toddschultz/session03/mailroom.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+global donors
+donors = ["Baby Huey", 1123.00, 456.00, 1789.00, "Mighty Mouse", 99.99, "Fred Flintstone", 5550.00, 5555.00, "Road Runner", 199999.00, "Papa Smurf", 1001.00, 1002.00, 1003.00]
+
+def prompt():
+    action = input("Would you like to (S)end a Thank You, (C)reate a Report or (Q)uit? ")
```

Comment:
you could add `lower()` at the end to avoid calling it each time:
`action = input("").lower()`